extension/popup/set_view_mod.test.js

```diff
@@ -0,0 +1,27 @@
+const restoreOptions = require('./reset');
+
+describe('test reset options function of popup code', () => {
+  const defaultId = 'text_crossed';
+  const selector = `#${defaultId}`;
+
+  beforeEach(async () => {
+    // add input to select
+    document.body.innerHTML = `<input type="radio" id="${defaultId}" >`;
+  });
+
+  // test using WebExtension mock
+  it('reset should call browser storage API', () => {
+    restoreOptions();
+    expect(browser.storage.sync.get).toHaveBeenCalled();
```

Comment:
This uses a mock version of the web extensions that can be used to check for calls.